presto-spi/src/main/java/com/facebook/presto/spi/security/AccessDeniedException.java

```diff
@@ -273,6 +274,16 @@ public static void denySetCatalogSessionProperty(String propertyName)
         throw new AccessDeniedException(format("Cannot set catalog session property %s", propertyName));
     }
 
+    public static void denySelectColumns(String tableName, Collection<String> columnNames)
+    {
+        denySelectColumns(tableName, columnNames, null);
+    }
+
+    public static void denySelectColumns(String tableName, Collection<String> columnNames, String extraInfo)
+    {
+        throw new AccessDeniedException(format("Cannot select from columns %s in table %s%s", columnNames, tableName, formatExtraInfo(extraInfo)));
```

Comment:
This can also be a view: `table` -> `relation`

Or we can say `in table or view %s` (might be better to avoid the term "relation" in user error messages)